python/pylibcudf/pylibcudf/io/csv.pyx

```diff
@@ -261,3 +267,97 @@ def read_csv(
         c_result = move(cpp_read_csv(options))
 
     return TableWithMetadata.from_libcudf(c_result)
+
+
+def write_csv(
+    SinkInfo sink_info,
```

Comment:
_Technically_ to match libcudf all of these parameters should be wrapped up in a `pylibcudf.io.csv.CsvWriterOptions` object that `write_csv` accepts.

I definitely this this is more "Pythonic" to spell out these options in the signature but not sure if you or others think we should align strictly with libcudf and create option objects

Reply:
> _Technically_ to match libcudf all of these parameters should be wrapped up in a `pylibcudf.io.csv.CsvWriterOptions` object that `write_csv` accepts.

Thanks, I just tried implementing everything necessary in `CudfWriterOptions` to do the migration

> I definitely this this is more "Pythonic" to spell out these options in the signature but not sure if you or others think we should align strictly with libcudf and create option objects

I left the signature as is for now until we decide how the API should look.